src/scripts/cronjobs/deleteSubsNoFxaUid.ts

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+/**
+ * This script deletes all rows in the "subscribers" table where fxa_uid is null/undefined/empty
+ *
+ * This is a one-off script that runs once in stage and prod
+ */
+
+import createDbConnection from "../../db/connect";
+
+// Make batch size mutable
+let BATCH_SIZE = 1000;
+
+/**
+ * This script deletes all rows in the "subscribers" table where fxa_uid is null/undefined/empty
+ */
+(async () => {
+  try {
+    console.log("Starting deletion of subscribers without fxa_uid...");
+    const knex = createDbConnection();
+
+    // Simple log helper
+    const log = (message: string) => {
+      const timestamp = new Date().toISOString();
+      console.log(`[${timestamp}] ${message}`);
+    };
+
+    // Get count before deletion
+    const beforeCountResult = await knex("subscribers")
+      .count("id as count")
+      .first();
+    // @ts-ignore: count type
+    const beforeCount = parseInt(String(beforeCountResult?.count || 0), 10);
```

Comment:
Could cast this to get rid of `@ts-ignore`:
```suggestion
    const beforeCount = Number((beforeCountResult?.count as string) || 0);
```

Reply:
still throws `Property 'count' does not exist on type 'string'.`